### ISSUE TITLE
Failure to Install TIFF Dependency

### ISSUE THREAD

**Issue by dinelanthony** [PixarAnimationStudios/USD#2862]
### Description of Issue
We are trying to build USD, and the script fails when trying to build the TIFF dependency. We can see that the file that the log file claims to be looking for is where it says it is so we are not sure why it is failing.

### Steps to Reproduce
1. Clone repo
2. Set up environment variables
3. Run C:\Users\PaoloTurri\Environments\usd\OpenUSD\build_scripts>python .\build_usd.py --build-monolithic --tests --docs --python-docs --debug-python --ptex --openvdb --embree --no-prman --openimageio --alembic --hdf5 --no-draco --materialx --opencolorio .\build

### System Information (OS, Hardware)
Windows 11

### Package Versions
USD 23.08
CMake 3.28.0
VS 15 2017

### Build Flags
--build-monolithic --tests --docs --python-docs --debug-python --ptex --openvdb --embree --no-prman --openimageio --alembic --hdf5 --no-draco --materialx --opencolorio

### Log File
[log.txt](https://github.com/PixarAnimationStudios/OpenUSD/files/13639580/log2.txt)

**Comment by jesschimein** [comment]
Filed as internal issue #USD-9073

**Comment by sunyab** [comment]
@dinelanthony From the logs, it looks like the libtiff build is looking for libm and picking up a version that's installed under `C:\wdc\Tools\lib\m.lib`. Do you know what that is? I'm wondering if it's a version of libm built in a mingw or Linux subsystem environment since Visual Studio doesn't seem to recognize that file's format.

I'm guessing that `C:\wdc\Tools\lib` path is getting searched during the build because it's on your PATH. If that's the case, you can try to tell the libtiff build to ignore that by adding the following argument when running build_usd.py:

```
build_usd.py ... --build-args TIFF,"-DCMAKE_IGNORE_PATH=C:\wdc\Tools\lib"
```

**Comment by FlorianZ** [comment]
@dinelanthony, closing this assuming that @sunyab 's guidance will get you unblocked here. Please, feel free to reopen or file a new issue if you still have questions!! Thank you.

**Comment by dinelanthony** [comment]
Leaving this closed but just commenting in case it could help anyone in the future:

We did not have a mingw or Linux subsystem for Windows installed, the additional build argument did not work however as recommended, editing our PATH to remove the problem directory fixed the issue. Thank you for all the help!
